flink-end-to-end-tests/test-scripts/common_kubernetes.sh

```diff
@@ -215,6 +215,8 @@ function cleanup {
     internal_cleanup
     kubectl wait --for=delete pod --all=true
     stop_kubernetes
+    set_config_key "rest.address" "localhost"
```

Comment:
isn't this already by `revert_flink_dir`?

Reply:
Ah yeah, I think you're right.